sdk/provisioning/Azure.Provisioning/tests/Infrastructure/OutputsSpanningModules/main.bicep

```diff
@@ -1,5 +1,8 @@
 targetScope = 'subscription'
 
+@description('Enable soft delete')
+param enableSoftDelete string
```

Comment:
why is this a param?

Reply:
I removed the location parameter which wasn't actually exposed since it was from an output, and instead added this one. location needs to be set via parameter or hardcoded.